api/users/serializers.py

```diff
@@ -1,7 +1,25 @@
 from rest_framework import serializers as ser
+from django.core.validators import URLValidator
 
 from api.base.serializers import JSONAPISerializer, LinksField, Link
 
+from website.models import ApiOAuth2Application, User
+from website.util.sanitize import strip_html
+
+def user_validator(user):
+    """
+    Raise a validation error if this is not a user object (or a valid user ID)
+    """
+    if isinstance(user, User):
+        return True
+    elif isinstance(user, basestring):
+        try:
+            User.load(user)
+        except:
```

Comment:
Don't use naked excepts. Also, `StoredObject.load` does not raise an exception if the record is not found--it returns `None`.